python/lsst/validate/drp/matchreduce.py

```diff
@@ -249,9 +285,18 @@ def _loadAndMatchCatalogs(repo, dataIds, matchRadius,
                 print("Skipping this dataId.")
                 continue
 
-        # We don't want to put this above the first "if useJointCal block"
-        # because we need to use the first `butler.get` above to quickly
-        # catch data IDs with no usable outputs.
+        if doApplyExternalWcs:
+            try:
+                wcs = butler.get("%s_wcs" % (externalWcsName), vId)
+            except (FitsError, dafPersist.NoResults) as e:
+                print(e)
```

Comment:
Do we have a logger here? I know the `print`s were pre-existing, but it would be good to replace them with `log.warn`s.

Reply:
I don't think we have a logger, and I'm not sure how to get one ... I hope it's okay if I punt on this as out of scope?

Reply:
Right this is all outside of a Task. Getting this to use a proper logger should happen, but it is out of scope for you.